arcaflow_plugin_stressng/stressng_plugin.py

```diff
@@ -139,6 +159,48 @@ def to_jobfile(self) -> str:
         return result
 
 
+@dataclass
+class HDDStressorParams:
+    stressor: str
+    hdd: int = field(
+        metadata={
+            "name": "HDD workers",
+            "description": (
+                "start N workers continually writing, "
+                "reading and removing temporary files"
+            ),
+        }
+    )
+
+    hdd_bytes: str = field(
+        metadata={
+            "name": "Bytes per worker",
+            "description": (
+                "write  N  bytes for each hdd process, the default is 1 GB. "
+                "One can specify the size in units of Bytes, KBytes, "
+                "MBytes and GBytes using the suffix b, k, m or g."
+            ),
+        }
+    )
+
+    hdd_write_size: str = field(
+        metadata={
+            "name": "Write Size",
+            "description": (
+                "specify size of each write "
+                "in bytes. Size can be from 1 byte to 4MB"
```

Comment:
Is this using the same unit suffix as the other fields?
We need to always assume that users are looking at this separately, without the context of the others.